x-pack/plugins/monitoring/public/services/enable_alerts_modal.js

```diff
@@ -12,12 +12,10 @@ export function enableAlertsModalProvider($http, $window, $injector) {
     const modalHasBeenShown = $window.sessionStorage.getItem('ALERTS_MODAL_HAS_BEEN_SHOWN');
     const decisionMade = $window.localStorage.getItem('ALERTS_MODAL_DECISION_MADE');
 
-    if (Object.keys(alerts.allAlerts).length > 0) {
+    if (Object.keys(alerts) > 0) {
```

Comment:
should be 
```suggestion
    if (Object.keys(alerts).length > 0) {
```

Reply:
Good catch! That was caused by a last-minute change 🙈